README.md

```diff
@@ -1 +1,52 @@
 # rds-poc
+
+POC of remote store in Terraform with RDS and EC2 instances
+
+**Diagram**
+
+![Diagram](diagram.png)
+
+**Pre-Requisites**
+
+* Configure your `AWS_ACCESS_KEY_ID` and your `AWS_SECRET_ACCESS_KEY`
+```
+ export AWS_ACCESS_KEY_ID=XXXXXXXXX
+ export AWS_SECRET_ACCESS_KEY=XXXXXXXXX
+```
+* Create your Key Pair on the region where is going to be deployed the solution
+
+**Steps**
+* On bucket_s3 directory:
+  - Update the archive "vars.tf" with your bucket name to store .tfstate
+
+  - After, you can run the following commands:
+```
+    $ terraform init
+    $ terraform plan
+    $ terraform apply
+```
+* On main directory:
+  - Update the archive "vars.tf" to set the following variables:
```

Comment:
Editing the vars.tf file is not the best practice here. How can we do this better?